src/applications/personalization/profile360/tests/actions/paymentInformation.unit.spec.js

```diff
@@ -41,37 +43,62 @@ const teardown = () => {
 };
 
 describe('actions/paymentInformation', () => {
-  beforeEach(setup);
-  afterEach(teardown);
+  describe('when global `ga` is set up correctly', () => {
+    beforeEach(() => setup({ mockGA: true }));
```

Comment:
this is the only actual change in the existing tests: passing the `mockGA` flag to the `setup()` call